app/src/main/java/org/shadowice/flocke/andotp/Utilities/AppStarts.java

```diff
@@ -0,0 +1,25 @@
+package org.shadowice.flocke.andotp.Utilities;
+
+/**
+     * Distinguishes different kinds of app starts: <li>
+     * <ul>
+     * First start ever ({@link #FIRST_NAME})
+     * </ul>
+     * <ul>
+     * First start in this version ({@link #FIRST_TIME_VERSION})
+     * </ul>
+     * <ul>
+     * Normal app start ({@link #NORMAL})
+     * </ul>
+     *
+     * @author Gigabyte Developers Incorporated
+     * inspired by
+     * @author Emmanuel Nwokoma
+     *
+     */
+
+public class AppStarts {
+    public enum AppStart {
```

Comment:
This could just go in Constants.java

Reply:
Ya ya. Just didn't want to touch the core files yet since the What's New Activity is still being tested! 
Well, stuffs like this can be cleaned up by @flocke or u yourself!